app/decorators/lib/basic_lti/basic_outcomes_decorator.rb

```diff
@@ -28,18 +34,8 @@ def update_submission_with_best_score
     return unless @submission
     return if @submission.excused?
 
-    best_score = @submission.score
-    best_grade = @submission.grade
-    versions   = @submission.versions
-
-    versions.each do |version|
-      version_score = YAML.load(version.yaml).stringify_keys['score']
-      if version_score.to_f > best_score.to_f
-        best_score = version_score
-        best_grade = YAML.load(version.yaml).stringify_keys['grade']
-      end
+    if @current_score.present? && @current_score.to_f > @submission.score.to_f
+      @submission.update_columns({ score: @current_score, grade: @current_grade, published_grade: @current_grade, published_score: @current_score })
```

Comment:
Is there a reason we are skipping all validations/callbacks with `update_columns`?

Reply:
Existing code. Probably? :)

Reply:
Jarred's theory: We didn't want to send duplicate data to the pipeline?

Reply:
But yeah, right now it seems like the pipeline is not getting accurate data when a score goes down.